server/sonar-server/src/main/java/org/sonar/server/organization/OrganizationCreationImpl.java

```diff
@@ -252,4 +254,10 @@ private void addCurrentUserToGroup(DbSession dbSession, GroupDto group, int crea
       dbSession,
       new UserGroupDto().setGroupId(group.getId()).setUserId(createUserId));
   }
+
+  private void insertOrganizationMember(DbSession dbSession, OrganizationDto organizationDto, UserDto userDto) {
```

Comment:
what is this OrganizationCreation class? Is it similar to the Updater we had? I know you didn't create the class